charts/k8s-infra/values.yaml

```diff
@@ -132,9 +132,72 @@ presets:
       cpu: {}
       load: {}
       memory: {}
-      disk: {}
-      filesystem: {}
-      network: {}
+      disk:
```

Comment:
are we certain that these are unwanted metrics and exclusion of these are okay for the users?

Reply:
I mentioned this in the linked issue description. This is not me deciding what to ignore. We are ignoring what Prometheus already does https://github.com/prometheus/node_exporter/blob/e6a9cfbdcdaa21bf9676c6cd37bef8160227f423/collector/diskstats_linux.go#L38